client/src/pages/TeacherPanel.jsx

```diff
@@ -59,6 +61,44 @@ const TeacherPanel = props => {
                                         size="mini"
                                         color="grey"
                                     />
+                                    <Button
+                                        icon="plus"
+                                        floated="right"
+                                        size="mini"
+                                    />
+                                    <Input
+                                        type="file"
+                                        onChange={e => {
+                                            const file = e.target.files[0];
+                                            const upload = new tus.Upload(
+                                                file,
+                                                {
+                                                    endpoint:
+                                                        process.env
+                                                            .REACT_APP_RESOURCE_MANAGEMENT +
```

Comment:
here you are over the maximum size of a line

Reply:
I know. CI also failed